eng/Versions.props

```diff
@@ -226,6 +222,9 @@
     <MonoCecilPackageVersion>0.10.1</MonoCecilPackageVersion>
     <NewtonsoftJsonBsonPackageVersion>1.0.2</NewtonsoftJsonBsonPackageVersion>
     <NewtonsoftJsonPackageVersion>12.0.2</NewtonsoftJsonPackageVersion>
+    <!-- Begin: STOP!!! Razor need to reference the version of JSON that our HOSTS support. -->
```

Comment:
Is this a copy-paste of a comment from tooling? How does a dev know how/when to update this version?

Reply:
Yea this is from https://github.com/dotnet/aspnetcore-tooling/blob/master/eng/Versions.props#L126. The tooling folks are responsible for updating this manually controlled version.